### PR TITLE
TASK-1074 Fix event timestamps

### PR DESCRIPTION
**Pull request by MrCreosote**
Set event timestamps to the time of the actual event since they're indexed.

### CODE REVIEW

File: src/us/kbase/workspace/test/listener/NullListenerFactory.java [hunk 1]
```diff
@@ -42,105 +44,103 @@ private void print(final String s) {
...
-			print("createWorkspace " + id);
+		public void createWorkspace(long id, Instant time) {
+			print(String.format("createWorkspace %s %s", id, time));
 		}
 
 		@Override
-		public void cloneWorkspace(long id, boolean isPublic) {
-			print("cloneWorkspace " + id + " " + isPublic);
+		public void cloneWorkspace(long id, boolean isPublic, Instant time) {
+			print(String.format("cloneWorkspace %s %s", id, isPublic, time));
```
[the remark this answers is not in the document]
Comment (MrCreosote) [reply src/us/kbase/workspace/test/listener/NullListenerFactory.java]: Yep, I've fixed that on the tip of my local branch, which is way ahead of this branch.

File: src/us/kbase/workspace/test/listener/NullListenerFactory.java [hunk 2]
```diff
@@ -42,105 +44,103 @@ private void print(final String s) {
...
 		}
 
 		@Override
-		public void setWorkspaceMetadata(long id) {
-			print("setWorkspaceMetadata " + id);
+		public void setWorkspaceMetadata(long id, Instant time) {
+			print(String.format("setWorkspaceMetadata %s %s", id, time));
```
[the remark this answers is not in the document]
Comment (MrCreosote) [reply src/us/kbase/workspace/test/listener/NullListenerFactory.java]: Looks like 2 and 2 to me?

[the remark this answers is not in the document]
Comment (sychan) [reply src/us/kbase/workspace/test/listener/NullListenerFactory.java]: Yeah, sorry, not enough sleep last night.

**Comment by MrCreosote** [comment]
@scanon @sychan fixed the first issue locally, but I'd prefer not to branch locally in order to fix the issue for this PR. Since this is just a demo listener and isn't actually used (or even loaded) in the WS proper, can we merge?

**Pull request merged**